src/components/DecimalCapacity/styles.module.scss

```diff
@@ -1,3 +1,7 @@
 .intergerPart {
   font-size: 16px;
+
+  @media (width <= 1000px) {
```

Comment:
1000px is not a common breakpoint value. Consider using 750 or 1200 to reduce the use of specific values for future breakpoint refactoring.

Reply:
I just followed the rules before. I didn't know the conclusion, so I just copied this value.

Reply:
Typical breakpoints can be found here https://getbootstrap.com/docs/5.0/layout/breakpoints/

But personally I don't value it much because IMO it's just for general templates instead of responsive layouts.

Reply:
You can consider using sizes like 750 / 1024 / 1200 instead of 1000 for breakpoints. The 1000 breakpoint is expected to be refactored and removed in the future.

Reply:
There are plenty of places using this size, but I didn't know what happened before, so I just copied the value from previous commits. Should we change all of them or only this in this PR?

Reply:
I think it's OK to update the breakpoint globally.

Reply:
Tracked by https://github.com/Magickbase/ckb-explorer-public-issues/issues/489